services/orchestrator/package.json

```diff
@@ -13,8 +13,9 @@
     "lint-fix": "pnpm linter-base-config --fix",
     "lint-fix-all": "pnpm lint-fix .",
     "linter-base-config": "eslint --ext=js,ts",
-    "test": "nx run test-linter && nx run test-type && nx run test-unit && nx run test-circular",
+    "test": "nx run test-linter && nx run test-type && nx run test-unit && nx run test-circular && nx run test-integration",
```

Comment:
The `test` target of swarmion projects is only for test suites at the bottom of the test pyramid, ie. unit, lint, type, etc...
So let's keep `test-integration` out of it